cmakemodules/script_pcl.cmake

```diff
@@ -14,10 +14,11 @@ IF(NOT DISABLE_PCL)
 	# PCL library:
 	# --------------------------------------------
 	find_package(PCL COMPONENTS io common registration visualization segmentation surface QUIET)
-	if (PCL_FOUND)
+	if(PCL_FOUND AND PCL_IO_FOUND AND PCL_COMMON_FOUND AND PCL_REGISTRATION_FOUND AND PCL_VISUALIZATION_FOUND AND PCL_SURFACE_FOUND)
+
 		# Filter: It seems clang + c++11 fails to build against PCL (for clang <3.5):
 		if (${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang" AND NOT CMAKE_MRPT_CLANG_VERSION GREATER 34)
```

Comment:
mmm... nope I referred to linking errors I had in the past while building `mrpt-pbmap`. Errors went like: "symbol not found for  function-foo(std::[c++11]::string...)" (not exact format). 

I always just disabled support for PCL to avoid the problem, which **did seem to me*** to be related to PCL being compiled with non-C++11 std::string so the linker was having problems with different ABIs. 

I never worried too much about it, so this is the first time I googled for it and... found [this](https://gcc.gnu.org/wiki/Cxx11AbiCompatibility) which breaks my theory of why the linker fails (!!). 

Having decent ROS and PCL packages in the official repos will be great... was about time!